README.md

```diff
@@ -82,19 +82,25 @@ It can be either a relative or absolute path.
 ## Testing
 
 The SDK has integration tests which run against a real Chatkit server.
-In order to run them, you must provide Chatkit instance credentials in
-your Gradle `local.properties` (or as `-D` arguments to the VM executing
-the tests)
 
-*Important:* The tests may delete any and all resources associated with
-the instance you provide. Create a new one. Do not share it with any other
-use.
+Firstly you will need to install the
+[Spek](https://www.spekframework.org/setup-android/) plugin, and Spek Framework in
+Android Studio. To do this go to Android Studio -> Preferences -> Plugins ->
+search for Spek, and Spek Framework and install. If you open a Spek test file, you
+should now see green play buttons to run each test (or the file).
+
+In order for the tests to pass you must provide a Chatkit instance
+credential to the VM - to do this edit the run configurations -> select Spek
```

Comment:
```suggestion
credential to the VM — to do this edit the run configurations -> select Spek
```
dash as opposed to minus

Reply:
I don't see a Gradle option when I edit my run configuation? Could you show me where you see this so I can ensure what we're telling people to do makes sense? If it helps, I don't have any run configuration enabled for Gradle, I run `gradle test` using the gradle menu.